pyproject.toml

```diff
@@ -90,3 +90,13 @@ module = [
 ignore_errors = true
 warn_unused_ignores = false
 
+
+[tool.cibuildwheel]
+before-test = ""
+test-requires = "pytest pytest-xdist"
+# Need to change root directory, otherisw the plugin won't find setup.cfg
+test-command = "(cd {project}; pytest --trace-config;  pytest -n2 -q test --nomemory --notimingintensive)"
+
+build = "*"
+skip = "cp36-* pp*"
+test-skip = ""
```

Comment:
not sure if it would make more sense to specify these as env argument in the yaml file

Reply:
Either is acceptable, I _think_ they kind of guide you to put whatever config possible into the toml so the config is mostly reusable across GH actions/travis/circle/etc if you ever need to switch

Reply:
I had the opposite impression from here https://cibuildwheel.readthedocs.io/en/stable/options/#options-summary considering that they list only evn variables (they do tell you the convention but still, it seem that env variables are the main thing)

Reply:
This is just speculation again, but I think the environment variables are a carryover from before the toml config was a thing. In the docs they just state that any of the environment variables can be used as toml keys [in the first paragraph here](https://cibuildwheel.readthedocs.io/en/stable/options/#configuration-file)

From what I've seen (hasn't been a ton), the toml config seems to be preferred where possible. The part that I like is it lets you have a pretty complex config but easily run it on local

Reply:
> The part that I like is it lets you have a pretty complex config but easily run it on local

yes I did not know you could run it locally. since that's a possibility using the pyproject file seems the best choice